distribution/kernel/carbon-home/dbscripts/mssql.sql

```diff
@@ -349,9 +349,6 @@ CREATE TABLE UM_TENANT (
 			UNIQUE(UM_DOMAIN_NAME),
 			UNIQUE(UM_TENANT_UUID));
 
-IF EXISTS (SELECT NAME FROM SYSINDEXES WHERE NAME = 'INDEX_UM_TENANT_UM_DOMAIN_NAME')
-DROP INDEX UM_TENANT.INDEX_UM_TENANT_UM_DOMAIN_NAME
-CREATE INDEX INDEX_UM_TENANT_UM_DOMAIN_NAME ON UM_TENANT (UM_DOMAIN_NAME);
```

Comment:
is this expected here?

Reply:
Since the table already defines the column as unique, creating an additional index is unnecessary.